app/src/main/res/layout/layout_main.xml

```diff
@@ -1,342 +1,341 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
The only functional change to this file is that I added an `android:id` to a `LinearLayout`. There are so many deltas because I took the opportunity to run this through Android Studio's XML formatter